ui/app/components/mfa/mfa-login-enforcement-form.js

```diff
@@ -60,11 +60,19 @@ export default class MfaLoginEnforcementForm extends Component {
     this.fetchAuthMethods();
   }
 
+  // Helper to add objects to target which sets the value of the tracked array
+  // in a way that will update the view
```

Comment:
I think we can also use ember array here - [docs](https://guides.emberjs.com/release/upgrading/current-edition/tracked-properties/#toc_arrays) which might a way we want to consider tracking arrays moving forward

Reply:
Oh yes, I've seen that used! Wasn't sure if that was a latest best practice, so thanks for the docs and I will update this to use that!